support-frontend/assets/pages/digital-subscription-landing/components/paymentSelection/paymentSelection.jsx

```diff
@@ -15,15 +15,22 @@ import {
 } from './helpers/paymentSelection';
 // styles
 import './paymentSelection.scss';
+import { Annual } from 'helpers/billingPeriods';
 
 type PropTypes = {
   paymentOptions: Array<PaymentOption>,
+  orderIsAGift: boolean,
 }
 
-const PaymentSelection = ({ paymentOptions }: PropTypes) => (
-  <div className="payment-selection">
-    {
-        (paymentOptions.map(paymentOption => (
+const PaymentSelection = ({ paymentOptions, orderIsAGift }: PropTypes) => {
+  // The following line makes sure the Annual payment selection card is on the right hand side
+  const paymentOptionsList = paymentOptions[0].title === Annual ?
+    [paymentOptions[1], paymentOptions[0]] :
```

Comment:
This feels a bit brittle- if we ever had more than two payment options this would make one disappear. We can use a sort here to always keep 'Annual' on the right:

```js
// Spread operator because .sort is mutating
[...paymentOptions].sort((opt1, opt2) => {
	if (opt1.title === 'Annual') return 1;
})
```

Reply:
I agree it's brittle. I kind of hate sort but I like what you've done with it here!